deploy-web/src/components/dashboard/Dashboard.tsx

```diff
@@ -128,30 +128,30 @@ export const Dashboard: React.FunctionComponent<IDashboardProps> = ({ dashboardD
             number={
               <>
                 <span>
-                  <FormattedNumber value={uaktToAKT(dashboardData.now.dailyUAktSpent)} maximumFractionDigits={2} />
+                  <FormattedNumber value={udenomToDenom(dashboardData.now.dailyUUsdSpent)} maximumFractionDigits={2} />
                 </span>
-                <AKTLabel />
+                <USDLabel />
               </>
             }
-            text="AKT spent (24h)"
+            text="USD spent (24h)"
             tooltip="Last 24h"
```

Comment:
We could add a little description saying that it adds up the value in AKT + the USDC spent

Reply:
Good idea 👍